src/screens/GalleryScreen/GalleryScreen.tsx

```diff
@@ -79,7 +80,7 @@ export const GalleryScreen: React.FC = () => {
    * @param item is taken from FlatList and used to render video inside Video component.
    * @returns a Video component.
    */
-  const renderVideo = ({item}) => (
+  const renderVideo = ({item}: {item: VideoFile}) => (
```

Comment:
Good job. You can also refactor this by using generic types. It will make it reusable in both places and many more where this syntax is used. It's up to you if you want to implement this.

```tsx
//implementation
export interface FlatListItem<T> {
  item: T
}

//usage
const renderVideo = ({item}: FlatListItem<VideoFile>) => (
```